misc/python/materialize/checks/scenarios_upgrade.py

```diff
@@ -690,3 +780,53 @@ def actions(self) -> list[Action]:
                 )
 
         return actions
+
+
+class SelfManagedEarliestToLatestDirectUpgrade(Scenario):
+    """
+    Upgrade from the oldest Self-Managed version in the supported window directly to the current version.
+    """
+
+    def __init__(
+        self,
+        checks: list[type[Check]],
+        executor: Executor,
+        features: Features,
+        seed: str | None = None,
+    ):
+        self.self_managed_versions = get_supported_self_managed_versions()
+        super().__init__(checks, executor, features, seed)
+
+    def base_version(self) -> MzVersion:
+        return self.self_managed_versions[0]
+
+    def actions(self) -> list[Action]:
+        versions = [self.base_version(), None]
+
+        print_upgrade_path(versions)
```

Comment:
Instead of
> Upgrading through versions ['v0.147.0', 'v0.147.3', 'v0.147.15', 'v0.147.17', 'v0.147.18', 'current']

I'd prefer

> Upgrading through versions v0.147.0 -> v0.147.3 -> v0.147.15 -> v0.147.17 -> v0.147.18 -> current